client/my-sites/stats/hooks/use-should-gate-stats.ts

```diff
@@ -148,6 +150,7 @@ const basicStats = [
 	STATS_FEATURE_INTERVAL_DROPDOWN_WEEK,
 	STATS_FEATURE_INTERVAL_DROPDOWN_MONTH,
 	STATS_FEATURE_INTERVAL_DROPDOWN_YEAR,
+	STATS_FEATURE_SUMMARY_LINKS_30_DAYS,
```

Comment:
Should we keep the `30 days` option usable as it was for sites created before the `stats/paid-wpcom-v3` release?

Reply:
Related thread? p1734000338014269-slack-C0438NHCLSY / https://github.com/Automattic/wp-calypso/pull/97431

Reply:
```suggestion
```